notebooks/244-code-language-id/244-code-language-id.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>I'm not sure that this image is relevant as there is no convolutions in transformers architecture :) Probably it required some proper description, what it show and which process illustrated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I would like to add something "The image bellow demonstrates general post-training quantization flow" before image, there is still not clear that text after image belongs to it as there is no any explicit logical connection between text and image</p><p> </p><p>Or possibly to remove it at all for avoiding confusion</p>

Reply:
@zilto, this the last my small comment, please address, resolve conflicts moving notebook on 245 position, fix typos detected by spellcheck and we are ready to merge it

Reply:
I removed the figure from the notebook and moved the folder and file to 245. For typos:
- Golang: false positive
- Javascript -> JavaScript
- TypeScript: false positive
- superset: false positive
- Husain: false positive
- prerequesites -> prerequisites
- Gb: false positive

I believe everything is in order. Let me know if further changes are necessary

Reply:
How do I add those as part of my PR ? This file doesn't exist on my local version of the repo. Not sure how to proceed

Reply:
@eaidova bump